src/index.js

```diff
@@ -0,0 +1,23 @@
+import './styles/main.scss';
+import { populateScoresList, refresh } from './util.js';
+import { postScore } from './scores.js';
+
+const gameId = 'zDajvCCZWwCARahxB4k3';
+let scoresArr = [];
+populateScoresList(gameId).then((res) => {
+  scoresArr = res;
+});
+
+document.getElementById('refresh-btn').addEventListener('click', () => {
+  refresh(scoresArr);
+});
```

Comment:
- [ ] When the `refresh-btn` is clicked, it should hit the GET API endpoint causing the scores list to change. Please implement this requirement because it this button doesn't work as expected in this moment 👍

Reply:
Fixed.